server/ctrlsubsonic/ctrl.go

```diff
@@ -93,7 +94,7 @@ func New(dbc *db.DB, scannr *scanner.Scanner, musicPaths []MusicPath, podcastsPa
 	chain := handlerutil.Chain(
 		withParams,
 		withRequiredParams,
-		withUser(dbc),
+		c.withUser,
```

Comment:
we don't need to make this a method of controller any more

just `withUser(dbc)` like it was before, and withUser can look like 

```
func withUser(dbc *db.DB) handlerutil.Middleware {
    return func(next http.Handler) http.Handler {
        return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

(see the previous implementation)

Reply:
Fixed in 6eddc2a1a9b2f1285cba4f757078e7f83627e818.  Thanks for the review!